components/zcash_protocol/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "zcash_protocol"
 description = "Zcash protocol network constants and value types."
-version = "0.5.3"
+version = "0.5.4"
```

Comment:
The workspace version in `Cargo.toml` needs to be bumped to this, as the new feature is being used in `zcash_primitives`.